### PR TITLE
Markdown conversion for `pt-BR` - Replace - API section :warning:  Do not squash :warning: 

### LINKED ISSUE #7220: Convert pt-br/web/api/setinterval to MD
Part of 100% Markdown Hackday

### LINKED ISSUE #7229: Convert pt-BR/docs/web/api/animationevent/elapsedtime to MD
Part of 100% Markdown Hackday

### LINKED ISSUE #7232: Convert pt-br/web/api/animationevent/pseudoelement to MD
Part of 100% Markdown Hackday

### LINKED ISSUE #7235: Convert pt-br/web/api/animationevent/animationevent to MD
Part of 100% Markdown Hackday

### LINKED ISSUE #7237: Convert pt-br/web/api/animationevent/animationname to MD
Part of 100% Markdown Hackday

### LINKED ISSUE #7241: Convert pt-BR/docs/web/api/batterymanager/dischargingtimechange_event to MD
Part of 100% Markdown Hackday

### LINKED ISSUE #7248: Convert pt-BR/docs/web/api/batterymanager/dischargingtime to MD
Part of 100% Markdown Hackday

### LINKED ISSUE #7252: Convert pt-BR/docs/web/api/keyboardevent to MD
Part of 100% Markdown Hackday

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of https://github.com/mdn/translated-content/pull/7961
Conversion report: https://gist.github.com/SphinxKnight/e66a41050cc8314ab9fcfd527ef8201f#file-md-conversion-problems-report-2022-09-15t05_08_30-471z-md

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/pt-br/web/api/document/close/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: Document.close()
+slug: Web/API/Document/close
+translation_of: Web/API/Document/close
+---
+{{APIRef("DOM")}}
+
+O metodo `document.close()` termina a gravação em um documento, aberto com [document.open()](/en/DOM/document.open "en/DOM/document.open").
```
Comment (josielrocha) [inline files/pt-br/web/api/document/close/index.md:8]:
```suggestion
O método **`document.close()`** termina a gravação em um documento, aberto com [document.open()](/en/DOM/document.open "en/DOM/document.open").
```

**Review by josielrocha — CHANGES_REQUESTED** [review]
Hi @SphinxKnight,
Thank you for your precious contribution.

I have made some suggestions and I would appreciate if you could check them.
Thank you a lot!

**Comment by SphinxKnight** [comment]
> Hi @SphinxKnight, Thank you for your precious contribution.
> 
> I have made some suggestions and I would appreciate if you could check them. Thank you a lot!

I would apply the ones related to "markdown-conversion" related issues. Otherwise, I think it would be best to address the rest later (ex. https://github.com/mdn/translated-content/pull/8517#discussion_r973519163 which was already an issue before the conversion). Moreover, since this PR's commits will not be squashed, that means introducing "more" commits to the main history.

I can file issues for each comment if you prefer. That way, we can fix them afterwards.

Please let me know, I don't want to block things and if you definitely prefer introducing those changes now, I'll comply :)

**Comment by josielrocha** [comment]
> > Hi @SphinxKnight, Thank you for your precious contribution.
> > I have made some suggestions and I would appreciate if you could check them. Thank you a lot!
> 
> I would apply the ones related to "markdown-conversion" related issues. Otherwise, I think it would be best to address the rest later (ex. [#8517 (comment)](https://github.com/mdn/translated-content/pull/8517#discussion_r973519163) which was already an issue before the conversion). Moreover, since this PR's commits will not be squashed, that means introducing "more" commits to the main history.
> 
> I can file issues for each comment if you prefer. That way, we can fix them afterwards.
> 
> Please let me know, I don't want to block things and if you definitely prefer introducing those changes now, I'll comply :)

It sounds good! Let's fix that after Markdown migration.

**Review by josielrocha — APPROVED** [review]

File: files/pt-br/web/api/angle_instanced_arrays/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - WebGL extension
+translation_of: Web/API/ANGLE_instanced_arrays
+---
+{{APIRef("WebGL")}}
+
+A extenção **`ANGLE_instanced_arrays`** é parte do [WebGL API](/pt-BR/docs/Web/API/WebGL_API) e permite desenhar o mesmo objeto, ou grupos de objectos semelhantes várias vezes, se eles compartilham os mesmos dados de vértice, contagem primitiva e tipo.
+
+As extensões WebGL estão disponíveis usando o método {{domxref("WebGLRenderingContext.getExtension()")}}. Para mais informações, veja também [Usando Extenções](/pt-BR/docs/Web/API/WebGL_API/Using_Extensions) no [WebGL tutorial](/pt-BR/docs/Web/API/WebGL_API/Tutorial).
+
+> **Nota:** **Disponibilidade:** Esta extenção só está disponível para contextos {{domxref("WebGLRenderingContext", "WebGL1", "", 1)}}. Em {{domxref("WebGL2RenderingContext", "WebGL2", "", 1)}}, a funcionalidade desta extenção está disponível no contexto WebGL2 por padrão e as constantes e métodos estao disponíveis sem o sufixo "`ANGLE`".
```
[the remark this answers is not in the document]
Comment (SphinxKnight) [reply files/pt-br/web/api/angle_instanced_arrays/index.md:17]: created as https://github.com/mdn/translated-content/issues/8627

**Comment by SphinxKnight** [comment]
@josielrocha sorry for the rush previously, I've re-read the comments and only one was an actual "content" update, I've filed an issue for it and applied the rest of the suggestions :) 

Thanks!

**Comment by SphinxKnight** [comment]
And I'll file an issue to check for patterns with `**` being escaped, I know we had to deal with some of them in `fr`

**Pull request merged**